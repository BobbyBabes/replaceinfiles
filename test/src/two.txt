hello world is the best way to learn
please protect the world, it's precious
