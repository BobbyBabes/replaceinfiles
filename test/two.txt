welcome world is the best way to learn
