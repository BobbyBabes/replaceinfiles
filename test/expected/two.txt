goodbye earth is the best way to learn
please protect the earth, it's precious
